src/plugins/api/score.h

```diff
@@ -89,13 +89,13 @@ class Score : public Ms::PluginAPI::ScoreElement
     /** Name of the score, without path leading to it and extension.\n \since MuseScore 3.2 */
     Q_PROPERTY(QString scoreName READ name WRITE setName)
     /** Number of measures (read only) */
-    Q_PROPERTY(int nmeasures READ nmeasures)
+    Q_PROPERTY(size_t nmeasures READ nmeasures)
     /** Number of pages (read only) */
-    Q_PROPERTY(int npages READ npages)
+    Q_PROPERTY(size_t npages READ npages)
     /** Number of staves (read only) */
-    Q_PROPERTY(int nstaves READ nstaves)
+    Q_PROPERTY(size_t nstaves READ nstaves)
     /** Number of tracks (#nstaves * 4) (read only) */
-    Q_PROPERTY(int ntracks READ ntracks)
+    Q_PROPERTY(size_t ntracks READ ntracks)
```

Comment:
I'm not sure if this will work, Qt may not support size_t when put to Qml.

Reply:
Leaving them `int` here...

Reply:
Hmm, there's an `ulonglong` available as a type on `Q_PROPERTY()`. Question is whether this is "platform-proof"

Reply:
Nope, that doesn't even compile

Reply:
with both, `size_t` and `int` it does